srcpkgs/CalculiX-cgx/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'CalculiX-cgx'
+pkgname=CalculiX-cgx
+version=2.20
+revision=1
+wrksrc=CalculiX
+_cgx_dir=cgx_${version}
+build_style="gnu-makefile"
+make_build_args="-C ${_cgx_dir}/src"
+makedepends="glu-devel libglvnd-devel libXmu-devel libXi-devel"
+depends="tetgen"
+short_desc="3D Structural Finite Element Program - Graphical interface"
+maintainer="Phicem <phicem@gmx.com>"
+license="GPL-2.0-or-later, custom:GLUT"
+homepage="http://www.dhondt.de/"
+distfiles="http://www.dhondt.de/cgx_${version}.all.tar.bz2"
+checksum=f4c840c3633718547e33f86b2d7bc6b35608dd8d6cbf9fa192cecef34f2dc908
+if [ "$XBPS_MACHINE" == "x86_64" ] ; then
+	hostmakedepends+="glibc-devel-32bit"
+fi
```

Comment:
```suggestion
```
not needed

Reply:
I added these lines during my attempts to cross-compile to armv7l (from an x86_64 machine). Without them, I get a compiler error about a missing header:
`fatal error: gnu/stubs-32.h: No such file or directory`

Adding glibc-devel-32bit fixes this issue, but is not enough to make compilation work, because then there is another error:
`/usr/include/math.h:462:21: error: '_Float128' is not supported on this target`

Since cross-compilation does not work, I guess you're right and these lines are not needed.